workflow/sample_files/base-enclosure-2stories-garage.xml

```diff
@@ -384,7 +386,7 @@
         <Doors>
           <Door>
             <SystemIdentifier id='Door1'/>
-            <AttachedToWall idref='Wall1'/>
+            <AttachedToWall idref='Wall3'/>

```

Comment:
@aspeake This change appears to be why the results are different on this branch relative to master. Previously the front door was on a wall between living space and outside, and now its on a wall between garage and outside. Can you look at why this regression occurred?

Reply:
@shorowit updated here: https://github.com/NREL/OpenStudio-HPXML/pull/863/commits/57acc1841d2db2601aef4353f092f490147b70e8